.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -48,3 +66,6 @@ The Rclone Developers
 #### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)
 
 
+
+#### The associated forum post URL from `https://forum.rclone.org`
```

Comment:
This should also probably be the first question or even the only question since the forum post already has the other requested information.

Reply:
So we visually increase forum importance. Agreed. (2)